### PR TITLE
added a unit test function

### PR DESCRIPTION
**Pull request by 0oHarsho0**
I have updated the **function.js** and **function.test.js** by adding an extra popOperation function so that we can perform unit test on popOperation as well.

### CODE REVIEW

File: function.test.js [hunk 1]
```diff
@@ -74,3 +70,23 @@ describe("removeElement", () => {
...
+    const result = popOperation(arr);
+    expect(result).toBe(3);
+  });
+
+  test("returns undefined for an empty array", () => {
+    const arr = [];
+    const result = popOperation(arr);
+    expect(result).toBeUndefined();
+  });
+});
```
Comment (ailaa-jadoo) [inline function.test.js:92]: you can add more test cases that are more of a edge cases where this can fail to show output.
